src/course_bot/quiz.clj

```diff
@@ -219,3 +219,8 @@
                    (do
                      (talk/send-text token id "Не понял, укажите корректный номер ответа (1, 2...).")
                      tx))))))
+
+(defn quiz-answ
+  "Function returns string with answers.
+  Parameter answers contains all answer of student."
+  [answers] (str "Quiz answers: " (str/join ", " answers)))
```

Comment:
Sorry, but DRY principle in this case completely ruin readability of the tests and also, mix test and non-test code.

Reply:
deleted them